public/template/messages.jade

```diff
@@ -10,9 +10,9 @@
       if _.isObject(activity)
         li
           small!= activity.content
-
-    if messages.items.length > 0
-      li.divider
-
+    if !messages.items.length
```

Comment:
Seems like this should probably be something like `messages.items.length === 0`?